### ISSUE TITLE
[one-component-per-file] do not check createApp function written by the developer

### ISSUE THREAD

**Issue by candy-Tong** [vuejs/eslint-plugin-vue#2201]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 8
- **eslint-plugin-vue version:** 9.14.1
- **Node version:** node16
- **Operating System:** win10

**Please show your full configuration:**

```json5

```

**What did you do?**	

```js
import { createApp } from 'other.js'
test(() => {
    createApp({})
})

test(() => {
    createApp({})
})
```
I write a function name `createApp` and when i write the unit test it has eslint `one-component-per-file` warning.

**What did you expect to happen?**

There is no eslint warning.

**What actually happened?**

It show the `one-component-per-file` warning`

**Repository to reproduce this issue**
